### ISSUE TITLE
Xiaomi Flore Care misst ab und zu falsche Erdfeuchte

### ISSUE THREAD

**Issue by Marty56** [AlCalzone/ioBroker.ble#395]
2-3 Mal am Tag wird eine Erdfeuchte von 160% übermittelt.
Es wäre wünschenswert, wenn solche Fehlmessungen vom PlugIn gefiltert würden.

**Comment by capitaenz** [comment]
Bei mir ist es ähnlich, hin und wieder wird 0% erkannt.

**Comment by AlCalzone** [comment]
Die große Frage ist hierbei, wie Fehlmeldungen zuverlässig erkannt werden können, ohne korrekte Werte fälschlicherweise zu ignorieren. 160% ist klar weil es außerhalb des sinnvollen Wertebereichs liegt, aber 0% Feuchte ist durchaus denkbar.

**Comment by Marty56** [comment]
Ja, ist nicht ganz trivial. Ich fange die 160% ab, dass ist natürlich einfach.

Man könnte eine Regel einführen, dass Werte, die mehr als 4% vom Vorgänger abweichen, mindestens drei mal empfangen werden, bis sie übernommen werden.

**Comment by capitaenz** [comment]
ganz genau kann ich nicht sagen wie es sich mit den 0% verhält. Ich bekomme ein Telegram auf mein Handy mit 0% und nächsten Morgen ist die Bodenfeuchte wieder bei ~10%.
Oder könnte es am Empfang liegen? Habe 3 Sensoren und die haben -90dB, -96dB und -88dB (1.OG in den Keller).
